content/logs/index.md

```diff
@@ -18,7 +18,7 @@ Datadog's Logs is currently available via public beta. You can apply for inclusi
 ## Getting started with the Agent
 
 Log collection requires an Agent version >= 6.0. Older versions of the Agent do not include the `Log collection` interface that we'll be using.
-If you are not using it already, please follow the installation instructions [here](https://github.com/DataDog/datadog-agent/blob/master/docs/beta/upgrade.md). We highly recommend to do a fresh install instead of the upgrade. 
+If you are not using it already, please follow the installation instructions [here](https://github.com/DataDog/datadog-agent/blob/master/docs/beta/upgrade.md). We highly recommend to do a fresh install instead of the upgrade.
```

Comment:
"We highly recommend to do a fresh install instead of the upgrade."

Please remove this line.

Reply:
If we wanted to keep it, it'd need to be:
"We highly recommend you perform a fresh install instead of the upgrade."

but in general upgrades are fine and I'd rather we not discourage them here.

@masci

Reply:
I do agree upgrades are ok, "highly recommend" seems a bit strong to me, it's like shouting upgrades don't work.

Reply:
👍 Removed